backend/src/middleware/handleHtmlContent/handleContentData.spec.js

```diff
@@ -359,25 +354,17 @@ describe('Hashtags', () => {
 
       describe('afterwards update the Post by removing a Hashtag, leaving a Hashtag and add a Hashtag', () => {
         // The already existing Hashtag has no class at this point.
-        const updatedPostContent =
+        const content =
           '<p>Hey Dude, <a class="hashtag" href="/search/hashtag/Elections">#Elections</a> should work equal for everybody!? That seems to be the only way to have equal <a href="/search/hashtag/Liberty">#Liberty</a> for everyone.</p>'
-        const updatePostMutation = gql`
-          mutation($postId: ID!, $postTitle: String!, $updatedPostContent: String!) {
-            UpdatePost(id: $postId, title: $postTitle, content: $updatedPostContent) {
-              id
-              title
-              content
-            }
-          }
-        `
 
         it('only one previous Hashtag and the new Hashtag exists', async () => {
           await mutate({
             mutation: updatePostMutation,
             variables: {
-              postId,
-              postTitle,
-              updatedPostContent,
+              id,
+              title,
+              content,
+              categoryIds,
```

Comment:
@Tirokk, I changed the variable names for all these to reuse the mutation and variables and DRY it out a bit
I think it's clear that it's a post id, title, etc... since we are using the `CreatePost`/`UpdatePost` mutation/variables, but I could change this, if you think it's better.